File: WebApp/Models/Line.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.Collections.Generic;
using WebApp.Persistence.Models;

namespace WebApp.Models
{
    public class Line
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public int Number { get; set; }

        [JsonConverter(typeof(StringEnumConverter))]
        public LineType LineType { get; set; }

        public List<Station> Stations { get; set; }

        public List<string> Departures { get; set; }
    }
}

File: Angular-app/src/app/pages/classes/line.ts
export class Line {
    Id: number;
    Name: string;
    Number: number;
    LineType: string;
    DepartureTime: string[];

    constructor(line: Line) {
        this.Id = line.Id;
        this.Name = line.Name;
        this.Number = line.Number;
        this.LineType = line.LineType;
        this.DepartureTime = line.DepartureTime;
    }
}
